lib/ansible/template/__init__.py

```diff
@@ -673,7 +708,7 @@ def do_template(self, data, preserve_trailing_newlines=True, escape_backslashes=
 
             try:
                 res = j2_concat(rf)
-                if new_context.unsafe:
+                if hasattr(new_context, 'unsafe') and new_context.unsafe:
```

Comment:
Wouldn't `getattr(new_context, 'unsafe', False)` be cleaner? (no idea about ansible code guidelines, just staw this PR when referenced in the related jinja PR

Reply:
yep, thanks